Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentTransform.h

```diff
@@ -27,7 +27,9 @@ class WindowComponentTransform : public ComponentWindow
 	bool translationModified;
 	bool rotationModified;
 	bool scaleModified;
+	char modifiedScaleAxis;
```

Comment:
This is just a suggestion and it's most definetly not needed for this feature, but in general, I prefer using enums in these instances, for a double purpose: one, you make sure a wrong value is never introduces, and two, whoever is reading your code can easily identify which are the accepted values.
```suggestion
	Axis modifiedScaleAxis;
```
Again, not needed and just personal preference, feel free to ignore this.